app/models/alchemy/page/page_scopes.rb

```diff
@@ -21,7 +21,7 @@ module Page::PageScopes
       # All pages locked by given user
       #
       scope :all_locked_by, ->(user) {
-        all_locked.where(locked_by: user.id)
+        all_locked.where(locked_by: user.send(Alchemy.user_class.primary_key))
```

Comment:
This is error prone: If the `user_class` does not `respond_to?(:primary_key)` we would get an exception.
Maybe something like:

```ruby
scope :all_locked_by, ->(user) {
  if Alchemy.user_class.respond_to? :primary_key
    all_locked.where(locked_by: user.class.primary_key)
  end
}
```